src/main/java/teammates/storage/api/FeedbackResponsesDb.java

```diff
@@ -473,25 +484,50 @@ private FeedbackResponse getFeedbackResponseEntity(
     }
 
     private List<FeedbackResponse> getFeedbackResponseEntitiesForQuestionInSection(
-                String feedbackQuestionId, String section) {
+                String feedbackQuestionId, String section, String sectionDetail) {
         List<FeedbackResponse> feedbackResponses = new ArrayList<>();
+        if ("BOTH".equals(sectionDetail)) {
+            feedbackResponses.addAll(load()
+                    .filter("feedbackQuestionId =", feedbackQuestionId)
+                    .filter("giverSection =", section)
+                    .filter("receiverSection =", section)
+                    .list());
+
+            // show responses in section with giver but without recipient
```

Comment:
Why in "BOTH" means "None" should be included?

Reply:
I followed the original implementation before this PR. For cases when it is directed to a team, the section selected will be "none". Somewhat, "none" can belong to any section.

Since "none" has no specific section, should I change it to implement for all cases instead?

Reply:
Before this PR, when section is selected, we only show response which has `giverSection = section` right?

Reply:
yes that is true. my bad... So should I add "none" to all filters of sectionDetails?